### PR TITLE
Remove .universal-login-default selectors

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-react/src/ui/Modals/ModalWrapper.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const ModalWrapper = ({children, modalClassName, hideModal, message}: Mod
 
   return (
     <div className={useClassFor('modal-backdrop')}>
-      <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login universal-login-default'}>
+      <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login'}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Modals/ModalWrapper.tsx:32]: Do we use modalClassName anywhere?

File: universal-login-react/src/ui/styles/themes/Jarvis/modalThemeJarvis.sass [hunk 2]
```diff
@@ -68,9 +68,6 @@
     color: #FFFFFF
     line-height: 19px
 
-  .universal-login-default
-    height: 100%
-
     @media only screen and (max-width: 770px)
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Jarvis/modalThemeJarvis.sass:71]: Shouldn't we remove it also?

File: universal-login-react/src/ui/Modals/ModalWrapper.tsx (hunk 1 above)
Comment (JustCaptcha) [reply universal-login-react/src/ui/Modals/ModalWrapper.tsx:32]: Yes, we are using it in LocalStorageBlockedWarningScreen, DashboardModal.

File: universal-login-react/src/ui/styles/themes/Jarvis/modalThemeJarvis.sass (hunk 2 above)
Comment (JustCaptcha) [reply universal-login-react/src/ui/styles/themes/Jarvis/modalThemeJarvis.sass:71]: I actually removed it.

Comment (JustynaBroniszewska) [reply universal-login-react/src/ui/styles/themes/Jarvis/modalThemeJarvis.sass:71]: No, `@media only ...` is not removed 😉

**Pull request closed without merging**